apps/angular/projection/src/app/ui/card.component.ts

```diff
@@ -0,0 +1,65 @@
+import { NgTemplateOutlet } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Directive,
+  Input,
+  TemplateRef,
+  contentChild,
+  input,
+  output,
+} from '@angular/core';
+
+interface ItemContext<T> {
+  $implicit: T;
+  item: T; // Required to support *options syntax as that does not use $implicit
+}
+
+@Directive({
+  selector: 'ng-template[appItem]',
+  standalone: true,
+})
+export class ItemDirective<T> {
+  @Input({ required: true }) appItem!: T[];
+
+  static ngTemplateContextGuard<TContext>(
+    dir: ItemDirective<TContext>,
+    ctx: unknown,
+  ): ctx is ItemContext<TContext> {
+    return true;
+  }
+}
+
+@Component({
+  selector: 'app-card',
+  template: `
+    <ng-content select="img"></ng-content>
```

Comment:
👍
you can also use the short synthax <ng-conent />

Reply:
What is the short syntax for   `<ng-content select="img"></ng-content>
` ?